src/dvclive/studio.py

```diff
@@ -14,7 +14,7 @@
 
 
 def _get_unsent_datapoints(plot, latest_step):
-    return [x for x in plot if int(x["step"]) > latest_step]
+    return [x for x in plot if int(x["step"]) >= latest_step]
```

Comment:
Best solution I could think of to send CPU metrics to studio. Otherwise `latest_step` == the step they are logged at and nothing is sent to studio.

Reply:
After a quick thought, it is not a good solution for the problem.

Reply:
The real solution is probably close to the hack in the `log_metrics` method in the lightning logger. But I'm not 100% sure on why this hack was introduced.

Reply:
for info: https://github.com/iterative/dvclive/issues/477

Reply:
Still working on that ...